app/controllers/agenda/agendaitems/agendaitem/documents.js

```diff
@@ -338,9 +338,4 @@ export default class DocumentsAgendaitemsAgendaController extends Controller {
   async unmarkForSignature(piece) {
     await this.signatureService.unmarkDocumentForSignature(piece);
   }
-
-  @action
-  refresh() {
-    this.send('reloadModel');
-  }
```

Comment:
In some places the `refresh` action has been removed while it's still used in the template to pass as an argument to a component. Can you double check all removals of the `refresh` actions?

Reply:
fixing this one

Reply:
also `router` service was missing here.